ci/matlab.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.  They are
+# provided by a third-party and are governed by separate terms of service,
+# privacy policy, and support documentation.
+#
+# This is a basic workflow to help you get started with MATLAB actions
+# For more information see: https://github.com/matlab-actions/overview
+
+name: MATLAB
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Set up MATLAB on the GitHub hosted runner
+      - name: Set up MATLAB
+        uses: matlab-actions/setup-matlab@v1
+
+      - name: Run tests and generate artifacts
+        uses: matlab-actions/run-tests@v1
+        with:
+          source-folder: CODE
```

Comment:
@mw-hrastega : [Within the command-line syntax, use all uppercase text to indicate placeholder text or content that varies for each user, such as a user or repository name](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#:~:text=Within%20the%20command%2Dline%20syntax%2C%20use%20all%20uppercase%20text%20to%20indicate%20placeholder%20text%20or%20content%20that%20varies%20for%20each%20user%2C%20such%20as%20a%20user%20or%20repository%20name)

Reply:
This seems to be the guidelines for GitHub docs. I saw [this](https://github.com/actions/starter-workflows/blob/1d039e1607c98d9b9a705102315b43896ad10633/ci/dotnet-desktop.yml#L59) in one of the templates.

Reply:
Alright, lets keep all-caps unless someone else any flags

Reply:
The all-caps doesn't indicate to me that it should be replaced by the user (despite what GitHub says). I prefer the something like "mysource".